test/e2e/validators_test.go

```diff
@@ -101,13 +101,15 @@ func validateBuildRunToSucceed(testBuild *utils.TestBuild, testBuildRun *buildv1
 	falseCondition := corev1.ConditionFalse
 
 	// Ensure the BuildRun has been created
-	err := testBuild.CreateBR(testBuildRun)
-	Expect(err).ToNot(HaveOccurred(), "Failed to create BuildRun")
+	if _, err := testBuild.GetBR(testBuildRun.Name); err != nil {
```

Comment:
Why should it already exist and be reused?

Reply:
This is caused by my Build/BuildRun-Prototype proposal that I would like to discuss with you actually. With this, the buildrun is already created and therefore this section needed a small update.